### ISSUE TITLE
MultiSelect.Filterable doesn’t support `useTitleInItem` (tooltip on hover)

### ISSUE THREAD

**Issue by hyungskim** [carbon-design-system/carbon-components-react#2328]
MultiSelect.Filterable from carbon-component-react doesn’t support `useTitleInItem` (tooltip on hover). It will show tooltip fine if we don’t use `Filterable`  option but this Filterable option is required for our UI search functionality. Can you add `useTitleInItem` functionality? It’s very crucial for Watson Knowledge Catalog UI

Version info:
carbon-components-react@6.97.0
carbon-components@9.79.0

<img width="272" alt="Screen Shot 2019-05-07 at 8 15 05 AM" src="https://user-images.githubusercontent.com/50377275/57320599-1a596080-70b4-11e9-9e08-c4dcdb6e02fb.png">

**Comment by asudoh** [comment]
Hi 👋 thank you for reporting! If you search for `useTitleInItem` in our `<MultiSelect>` code, you'll find that it simply translated to our `<Checkbox>`: https://github.com/carbon-design-system/carbon-components-react/blob/v7.1.0/src/components/MultiSelect/MultiSelect.js#L300. You can try doing so in your fork, and if it works well and you'll see it valuable, please be sure to contribute it back!

**Comment by hyungskim** [comment]
Hi @asudoh thanks for responding quick.  Yeah, I've noticed `useTitleInItem` in MultiSelect.js but not in FilterableMultiSelect.js.  FilterableMultiSelect.js seems to be the actual source code where we need `useTitleInItem`.

**Comment by asudoh** [comment]
You are right - Would you try applying the `<MultiSelect>` code to `<MultiSelect.Filterable>`? Thanks!

**Comment by hyungskim** [comment]
I love to apply my code change but I'm getting access denied error when trying to push my changes.  Can I request access?  

<img width="626" alt="Screen Shot 2019-05-07 at 7 07 35 PM" src="https://user-images.githubusercontent.com/50377275/57344351-c2dee300-70fb-11e9-9cfa-88fdb694b28c.png">

**Comment by asudoh** [comment]
Hi @hyungskim the process is creating your own fork and submit a PR based on that. Please see https://github.com/carbon-design-system/carbon-components-react/blob/v7.2.0/.github/CONTRIBUTING.md#setup for more details. Thanks!
